explorer/explorer.go

```diff
@@ -317,8 +317,9 @@ func (exp *explorerUI) Store(blockData *blockdata.BlockData, msgBlock *wire.MsgB
 	difficulty := blockData.Header.Difficulty
 	bdHeight := newBlockData.Height
 
-	// 248 blocks take almost a day to mine
-	last24hrDifficulty := exp.blockData.DifficultyAtHeight(bdHeight - 248)
+	// last 24hr timestamp
+	timestamp := time.Now().Add(-24 * time.Hour).Unix()
+	last24hrDifficulty := exp.blockData.DifficultyInLast24Hrs(timestamp)
```

Comment:
(The function itself doesn't consider 24 hours, just the time stamp.)

Reply:
will remove the 24hrs part and add a comment.